base/exports.jl

```diff
@@ -1233,6 +1233,7 @@ export
     @html_str,
     @doc,
     @doc_str,
+    docstrloc,
```

Comment:
Editing docstrings is mostly a developer thing to do, so I agree that it makes more sense not to export it.